DEMO.md

```diff
@@ -22,27 +22,35 @@ cd labfiles/arista-ansible/atd-avd
 
 In addition, open CloudVision Portal (CVP) by clicking the __CVP__ link. Then, log in with the username `arista` and the auto-generated password on the lab topology screen.
 
-## 2. Configure your inventory
+## 2. Configure your credential
 
-Because the password is auto-generated, you must update the `ansible_password` variable in this inventory file to use the correct credentials. Also, ensure the `ansible_host` variable for `cv_atd1` is set to the static IP below.
+Because the password is auto-generated, you must update the `ansible_password` variable. We will read a configuration file to set our credential file as an environment variable.
 
-```yaml
-# edit atd-inventory/inventory.yml
----
-all:
-  children:
-    cv_servers:
-      hosts:
-        cv_atd1:
-          ansible_host: 192.168.0.5
-    ...
-  vars:
-    ansible_user: arista
-    ansible_password: # Update password with lab credentials
-    ...
+```shell
+export LABPASSPHRASE=`cat /home/coder/.config/code-server/config.yaml| grep "password:" | awk '{print $2}'`
 ```
 
-## 3. Prepare ATD
+## 3. Install all the requirements
+
+The code block below performs the following actions:
+
+- Moves to the `labfiles` directory
+- Installs version 4.1.0 of the arista.avd collection
+- Exports an environment variable based on the location of the `arista.avd` collection
+- Leverages the new environment variable to install the Python requirements for arista.avd
+- Clones this repository
+- Moves to the cloned repository folder
+
+    ```shell
+    cd /home/coder/project/labfiles
+    ansible-galaxy collection install arista.avd:==4.1.0
+    export ARISTA_AVD_DIR=$(ansible-galaxy collection list arista.avd --format yaml | head -1 | cut -d: -f1)
+    pip3 install -r ${ARISTA_AVD_DIR}/arista/avd/requirements.txt
+    git clone https://github.com/arista-netdevops-community/atd-avd.git
+    cd atd-avd
```

Comment:
@JulioPDX please go ahead and merge it. Let's not wait for another review.  
Just make sure we also replace one-liner install script with the same installation procedure in the detailed guide.
There might be still a few things to fix, but we must merge it now as the current AVD-ATD state is completely broken.